CMakeLists.txt

```
@@ -26,6 +26,21 @@ target_compile_features(fplus INTERFACE cxx_std_14)
 find_package(Threads REQUIRED)
 target_link_libraries(fplus INTERFACE Threads::Threads)
 
+# ---- Create amalgamated library ----
+
+# This will create the amalgamated library into include_all_in_one/include/fplus/fplus.hpp
+find_package(Python COMPONENTS Interpreter QUIET)
+if (Python_FOUND)
```

```suggestion
if(Python_FOUND)
```

Since `if` is "just" a command, there should be no space after it.  
All good after this.